bottom/__init__.py

```diff
@@ -11,42 +13,66 @@
 class Client(event.EventsMixin):
     __conn_cls = connection.Connection
 
-    def __init__(self, host, port, *, encoding='UTF-8', ssl=True, loop=None):
+    def __init__(self, host: str, port: str, *, encoding: str = 'UTF-8',
+                 ssl: bool = True, loop: asyncio.BaseEventLoop = None) -> None:
```

Comment:
It's not really a `typing` thing. In python, `__init__` does indeed return None. it's `__new__` that returns the class instance.

You could sort of envision instance creation as such:

    instance = Class.__new__()
    instance.__init__()

Of course that's simplifying a lot, but all `__init__` methods should return None.